amnestywebsite/humanity-theme
# Replace Search and Search Results with FSE templates

Ref: https://github.com/amnestywebsite/humanity-theme/issues/373, https://github.com/amnestywebsite/humanity-theme/issues/385

Replaces the old search.php and searchpage.php templates with HTML equivalents, and the necessary patterns to contain the logic.

**Steps to test**:
1. Navigate to /search
2. Check everything looks correct, use the filters to filter the posts and make sure they work correctly
3. Clear all filters, and enter a search query
4. The results should display correctly
5. Use the filters again and check they filter the results
6. Navigate to the site editor and check the templates for the search and searchpage
7. They should match the front end

**Considerations**:
- Accessibility
  - Regions (semantic HTML tags, e.g. `<section>`; [landmarks](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#landmark_roles); etc.)
  - Screen reader compatibility, labels ([`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label), [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby), etc.)
  - Keyboard navigability
- Localisation
- RTL

Example: http://bigbite.im/v/Qyiwjq
